VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.scss

```diff
@@ -26,6 +26,26 @@
   }
 }
 
+.participant-grid {
+  color: rgba(255, 255, 255, 0.5);
+  font-size: 16px;
+  font-family: 'GDS Transport', arial, sans-serif;  
+  .participant-group-row{
+    display: flex;
+    border-bottom: 0.5px  rgba(255, 255, 255, 0.2);
+    padding-top: 1em;
+    padding-bottom: 0.2em;

```

Comment:
this looks like the only difference btw this class and `.participant-row` - i'd just apply `participant-row` to the element and add this class if needed, with just the adjusted `padding-bottom`